examples/disable-personal-account-javascript/src/main.js

```diff
@@ -92,9 +159,11 @@ function updateUserInfo() {
 
 async function updateOrgInfo() {
   const activeOrg = clerk.organization;
+  console.log("activeOrg", activeOrg);
```

Comment:
Do you think this would be useful for debugging? Just flagging in case it was commited by mistake